NetCoreDemos/Views/Toolbar/Multiline.cshtml

```diff
@@ -0,0 +1,337 @@
+                        .Location(ToolbarItemLocation.After)
+                        .LocateInMenu(ToolbarItemLocateInMenuMode.Auto);
+                        
+                    items
+                        .Add()
+                        .Widget(w => w
+                            .Button()
+                            .Icon("help")
+                            .Text("About")
+                            .OnClick("aboutButtonClick")
+                        )
+                        .ShowText(ToolbarItemShowTextMode.InMenu)
+                        .Location(ToolbarItemLocation.After)
+                        .LocateInMenu(ToolbarItemLocateInMenuMode.Auto);
+                }
+            )
+        )
+    )
+
+    <div class="options-container">
+        <div class="caption">Options</div>
+
+        @(Html.DevExtreme().RadioGroup()
+            .ID("toolbar-modes")
+            .Layout(Orientation.Horizontal)
+            .Items(toolbarModes)
+            .Value(toolbarModes[1])
+            .OnValueChanged("toolbarModesValueChanged"))
+    </div>
+</div>
+
+@using (Html.DevExtreme().NamedTemplate("separator-template")) {
+    <div class="toolbar-separator"></div>
+}
+
+@using (Html.DevExtreme().NamedTemplate("menu-separator-template")) {
+    <div class="toolbar-menu-separator"></div>
+}
+
+@using (Html.DevExtreme().NamedTemplate("text-align-template")) {
+    @(Html.DevExtreme().ButtonGroup()
+        .KeyExpr("Style")
+        .StylingMode(ButtonStylingMode.Text)
+        .Items(new [] {
+            new { icon = "alignleft", alignment = "left", hint = "Align left" },
+            new { icon = "aligncenter", alignment = "center", hint = "Center" },
+            new { icon = "alignright", alignment = "right", hint = "Align right" },
+            new { icon = "alignjustify", alignment = "justify", hint = "Justify" },
+        })
+        .SelectedItemKeys(new[] { "left" })
+        .OnItemClick("buttonGroupItemClick"));
+}
+
+@using (Html.DevExtreme().NamedTemplate("text-align-menu-template")) {
+    @(Html.DevExtreme().ButtonGroup()
+        .KeyExpr("Style")
+        .StylingMode(ButtonStylingMode.Text)
+        .Items(new [] {
+            new { icon = "alignleft", alignment = "left", hint = "Align left", text = "Align left" },
+            new { icon = "aligncenter", alignment = "center", hint = "Center", text = "Center" },
+            new { icon = "alignright", alignment = "right", hint = "Align right", text = "Align right" },
+            new { icon = "alignjustify", alignment = "justify", hint = "Justify", text = "Justify" },
+        })
+        .SelectedItemKeys(new[] { "left" })
+        .OnItemClick("buttonGroupItemClick"));
+}
+
+<script>
+    function toolbarModesValueChanged(data) {
+        var toolbar = $("#toolbar").dxToolbar("instance");
+        toolbar.option("multiline", data.value === toolbarModes[0]);
+
+        var toolbar-modesInstance = $("#toolbar-modes").RadioGroup("instance");
```

Comment:
I'm wondering how it worked ..
```suggestion
        var toolbarModesInstance = $("#toolbar-modes").RadioGroup("instance");
```

Reply:
typo

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/05c2a4c6f2f88d68152ae92b286c4eba0e1284b7